core/templates/dev/head/pages/exploration_player/ExtractImageFilenamesFromStateService.js

```diff
@@ -189,16 +185,13 @@ oppia.factory('ExtractImageFilenamesFromStateService', [
     var _getImageDimensionsInState = function(state) {
       var fileDimensions = {};
       if (state.interaction.id === INTERACTION_TYPE_IMAGE_CLICK_INPUT) {
-        var filepathObject = (
+        var filename = (
           state.interaction.customizationArgs.imageAndRegions.value.imagePath);
-        if (!filepathObject.width) {
-          var filename = filepathObject;
-          filepathObject = {
-            name: filename,
-            width: 500,
-            height: 200
-          };
-        }
+        var filepathObject = {
```

Comment:
Why do you even need this object? You might as well just assign 500, 200 directly below...

Reply:
Yeah, will do that.

Reply:
I don't think you've done that...

Reply:
Let's do it in a follow-up PR